docs/content/readme.md

```diff
@@ -47,14 +45,13 @@ See the [Getting Started setup guide](direct/get-started.md) for getting started
 
 ## Contributing
 
-
 We ❤️ our contributors! If you're interested in helping us out, please head over to our [Contributing](contribution-guidelines/contributing-inc.md) guide.
 
 ## Getting in touch
 
 There are several ways to communicate with us:
 
-Instantly get access to our documents and meeting invites [http://kubestellar.io/joinus](http://kubestellar.io/joinus)
+Instantly get access to our documents and meeting invites [http://kubestellar.io/joinus](http://kubestellar.io/joinus).
```

Comment:
This list item is still not formatted like the other items in the list.
This problem was not introduced in this PR, and does not have to be fixed in this PR.